aws/terraform/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Pangeo Data
```

Comment:
@yakutovicha Since I made changes, we might want/have to add a line here?
```suggestion
Copyright (c) 2020 Simon Adorf (EPFL)
Copyright (c) 2020 Pangeo Data
```